src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -140,4 +140,14 @@ public boolean equals(Object other) {
                 || (other instanceof UniquePersonList // instanceof handles nulls
                         && this.internalList.equals(((UniquePersonList) other).internalList));
     }
+
+    /**
+     * Sorts the internal list.
```

Comment:
a better way to describe this might be return a sorted version of the internal list, as the internal list is not modified here.